content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -51,18 +51,14 @@ To enable IPv4/IPv6 dual-stack, enable the `IPv6DualStack` [feature gate](/docs/
       * `--feature-gates="IPv6DualStack=true"`
       * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>` eg. `--cluster-cidr=10.244.0.0/16,fc00::/24`
```

Comment:
Should we also change the cluster CIDR example to `fc00::/48`?

Reply:
Hello @lachie83 
Reading through the changes, the usage examples get lost in the list of flags. The presentation of the information
would improve if you separated out the examples from the list of flags. A couple of _options_ come to mind:
-  Add a sub item under the flag with the usage example
-  Add a note shortcode (after the list) highlighting the default values for the flags
-  The docs style guide recommends using `For example` instead of `eg.`

Also,  is line 59 correct?
```
 * `--cluster-cidrs=<IPv4 CIDR>,<IPv6 CIDR>` 
```